drivers/postgres_test.go

```diff
@@ -0,0 +1,63 @@
+package drivers
+
+import (
+	_ "github.com/lib/pq"
+	"github.com/volatiletech/null"
+	"reflect"
+	"testing"
+)
+
+/*
+Added and commented for the purpose of testing as the values will vary in each local setup.
```

Comment:
If you want to keep this as a note or something like that, you should consider converting it to an [example](https://blog.golang.org/examples).